src/java.desktop/share/classes/java/awt/color/ICC_ProfileGray.java

```diff
@@ -140,10 +137,6 @@ public float getGamma() {
      *         table
      */
     public short[] getTRC() {
-    short[]    theTRC;
-
-        theTRC = super.getTRC(ICC_Profile.icSigGrayTRCTag);
-        return theTRC;
```

Comment:
hmm this old code looked like it might be trying to avoid its own override but no such override exists ...